pogrep.py

```diff
@@ -1,75 +1,164 @@
+                        fill(left, width=(WIDTH - 7) // 2),
                         fill(entry.msgstr, width=(WIDTH - 7) // 2),
                     ]
                 )
-    print(colorize(tabulate(table, tablefmt="fancy_grid"), pattern))
+    if not file_match:
+        print(colorize(tabulate(table, tablefmt="fancy_grid"), pattern, prefixes))
+
+
+def process_path(path, recursive, exclude_dir):
+    files = []
+    if len(path) == 0:
+        if recursive:
+            files = glob.glob("**/*.po", recursive=True)
+            if exclude_dir:
+                return [elt for elt in files if exclude_dir.rstrip(os.sep) + os.sep not in elt]
+            else:
+                return files
+        else:
+            sys.exit(0)
+    for elt in path:
+        if os.path.isfile(elt):
+            files.append(elt)
+        elif os.path.isdir(elt):
+            if recursive:
+                files.extend(glob.glob(elt + os.sep + "**/*.po", recursive=True))
+            else:
+                print("{}: {}: Is a directory".format(sys.argv[0], elt), file=sys.stderr)
+                sys.exit(1)
+        else:
+            print("{}: {}: No such file or directory".format(sys.argv[0], elt), file=sys.stderr)
+            sys.exit(1)
+    if exclude_dir:
+        files = [elt for elt in files if exclude_dir.rstrip(os.sep) + os.sep not in elt]
+    return files
 
 
 def parse_args():
     parser = argparse.ArgumentParser(description="Find translated words.")
+    parser.add_argument("-F", "--fixed-strings", action="store_true",
+                        help="Interpret pattern as fixed string, not regular expressions")
+    parser.add_argument("-i", "--ignore-case", action="store_true",
+                        help="Ignore case distinctions, so that characters that differ only in case match each other.")
+    parser.add_argument("-w", "--word-regexp", action="store_true",
+                        help="Select only those lines containing matches that form whole words.")
+    parser.add_argument('-n', "--line-number", action="store_true",
+                        help="Prefix each line of output with the 1-based line number within its input file.")
+    parser.add_argument("-l", "--files-with-matches", action="store_true",
+                        help="Suppress normal output; instead print the name of each input file from which output "
+                             "would normally have been printed.  The scanning will stop on the first match.")
+    parser.add_argument("-s", "--no-messages", action="store_true",
+                        help="Suppress error messages about nonexistent or unreadable files.")
+    parser.add_argument("-r", "--recursive", action="store_true",
+                        help="Read  all  files under each directory, recursively, following symbolic links only "
+                             "if they are on the command line.  Note that if no file operand is given, pogrep searches"
```

Comment:
```suggestion
                             "if they are on the command line.  Note that if no file operand is given, pogrep searches "
```

Else we're having `searchesthe working directory`.